corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -126,6 +127,14 @@ def _form_undelete(doc):
             doc['server_modified_on'] = json_format_datetime(datetime.utcnow())
         return _soft_undelete(XFormInstance.get_db(), form_ids, _form_undelete)
 
+    @staticmethod
+    def modify_attachment_xml_and_metadata(form_data, form_attachment_new_xml):
+        form_data.put_attachment(form_attachment_new_xml, name="form.xml", content_type='text/xml')
+        operation = XFormOperation(user_id=SYSTEM_USER_ID, date=datetime.utcnow(),
+                                   operation='gdpr_scrub')
+        form_data.history.append(operation)  # TODO: should this show in Form History tab? it doesn't
```

Comment:
is this comment still true?

Reply:
Nope, thanks for catching that
Removed here:
https://github.com/dimagi/commcare-hq/pull/19979/commits/39339c57c1a196f76b02b81d5f29ed0e4859a9b7